### DISCUSSION TITLE
n-gon parkour level

### DISCUSSION THREAD

**Discussion by tbxyd** [landgreen/n-gon#360] [Show and tell]
hey guys, I made an n-gon level and it uses movement techniques to beat. not too long, but can be a little hard
**techniques used:** corner clip, crouch jump

**instructions on playing the level:**
1. go to https://c-rxxp-y.github.io/n-gon-enhanced
2. the first level will always be the parkour one, so just start the game

**rules:**
1. no testing mode
3. no fields other than default (field emitter)

if y'all cant figure out the movement techs, I'll post how to do them

**reward**
if you can prove you beat it without cheating (submit a screen recording/screenshot documenting lack of cheating), then there will be a reward for the first person.
**i will add a custom skin tech to my mod to my mod based on the winners design**

play my mod (if you want) --> https://n-gon-enh.vercel.app

**Comment by AnnonymousNerd87** [comment]
this is legit

**Comment by AnnonymousNerd87** [comment]
I can do everything but the first jump segmented, but not continuous

**Reply by tbxyd** [reply]
first jump is corner clip, then the rest is crouch jumps and crouch resets

**Comment by AnnonymousNerd87** [comment]
Corner clip?

**Comment by tbxyd** [comment]
Basically you jump perfectly on a corner of the map and it boosts you forward farther than normal


Sent from my Apple Watch

**Comment by AnnonymousNerd87** [comment]
lemme guess, its (near) frame perfect

**Comment by AnnonymousNerd87** [comment]
I tried it multiple times

**Comment by tbxyd** [comment]
Not really, i can do it 40% of the time. Just takes practice


Sent from my Apple Watch

**Comment by AnnonymousNerd87** [comment]
I tried it like 20 times and didn't get it once

**Comment by AnnonymousNerd87** [comment]
btw I finally did it using testing to skip the first jump and then no
testing

**Comment by AnnonymousNerd87** [comment]
that screen recording should be it

**Comment by AnnonymousNerd87** [comment]
darn, message too large

**Comment by tbxyd** [comment]
[parkour.webm](https://github.com/user-attachments/assets/9ae19801-4eb6-44f9-9f6f-a97d25f9c757)

ignore the massive lag spike 💀

**Comment by AnnonymousNerd87** [comment]
ok, just really tight positioning

**Comment by AnnonymousNerd87** [comment]
wait nvm message too large

**Comment by AnnonymousNerd87** [comment]
Screen recording 2024-11-21 9.20.30 AM.webm
<https://drive.google.com/open?id=12TCymf6FxFRXhmYgJm3VehHc0NElcXaI>

**Comment by AnnonymousNerd87** [comment]
I beat it!

**Reply by tbxyd** [reply]
nice job! design a skin tech (for example nitinol or CPT symmetry) and its mechanics, and I'll add it!

new harder level may be coming soon...👀

**Comment by AnnonymousNerd87** [comment]
Thanks, I'll get back to you after thanksgiving and I've thought over it
with my brother

**Comment by AnnonymousNerd87** [comment]
minor wall jump bug found on your level

**Comment by tbxyd** [comment]
What is it? Can you just jump while sliding down a wall?


Sent from my Apple Watch

**Comment by tbxyd** [comment]
Oh btw you did leak your school district, city, and state in that screen recording


Sent from my Apple Watch

**Comment by AnnonymousNerd87** [comment]
while going fast enough, you can clip into wall that are inside other walls
for 1 frame, if you are holding w this will let you jump

**Comment by AnnonymousNerd87** [comment]
I'll delete it

**Comment by AnnonymousNerd87** [comment]
wait

**Comment by AnnonymousNerd87** [comment]
I'm not logged in, can you delete it?

**Comment by AnnonymousNerd87** [comment]
I just set it to restricted so only I can see the recording

**Comment by tbxyd** [comment]
Yeah its a collision glitch, once i furnish the level it'll be submitted for community maps and Ill try to fix it


Sent from my Apple Watch

**Comment by AnnonymousNerd87** [comment]
if you make this a community level it will need a rework so you can fit a
boss or two. also if you miss the final jump you have to restart the level
by jumping off, so maybe it might be a good idea to add steps back up to
the level bc if you can do the level you can do the final jump.

**Comment by corneeme** [comment]
bro its not hard...            
 bc u left testing mode on
XD XD

**Comment by AnnonymousNerd87** [comment]
no testing allowed for full completion

**Comment by AnnonymousNerd87** [comment]
we both did it without testing

**Comment by AnnonymousNerd87** [comment]
multiple times

**Comment by corneeme** [comment]
whelp im just bad then

**Comment by corneeme** [comment]
probably should've realized that

**Comment by tbxyd** [comment]
Its not hard, just takes some practice. I have another level coming this week that is way harder. It will seem impossible, but me and a few friends verified it


Sent from my Apple Watch

**Comment by dotbot123** [comment]
has anyone beaten this yet?

**Comment by AnnonymousNerd87** [comment]
yes, I and c.rxxp.y have multiple times. I haven't done the updated one tho
lol
